packages/postgres-database/src/simcore_postgres_database/models/projects_to_folders.py

```diff
@@ -0,0 +1,49 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .folders_v2 import folders_v2
+
+projects_to_folders = sa.Table(
+    "projects_to_folders",
+    metadata,
+    sa.Column(
+        "folder_id",
+        sa.BigInteger,
+        sa.ForeignKey(
+            folders_v2.c.folder_id,
+            name="fk_projects_to_folders_to_folders_id",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
+        ),
+    ),
+    sa.Column(
+        "project_uuid",
+        sa.String,
+        sa.ForeignKey(
+            "projects.uuid",
+            name="fk_projects_to_folders_to_projects_uuid",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
+        ),
+    ),
+    sa.Column(
+        "user_id",
```

Comment:
What does the user_id has to do in a projects_to_folders table? i would not have expected it.

Reply:
Its important, because if you are in your personal workspace, basically the current Dashboard view. You can manage your projects and put them in your folder. Therefore for example the same project P, may live in 2 different folders. Private folder of User 1 and Private folder is User 2.